File: 백준/Bronze/5522. 카드 게임/카드 게임.java
import java.io.*;

class Main {
    static int nextInt() throws IOException {
        int c, n = System.in.read() & 15;
        while ((c = System.in.read()) > 32) {
            n = (n << 3) + (n << 1) + (c & 15);
        }
        return n;
    }
    
    static final int TURN = 5;
    
    public static void main(String[] args) throws IOException {
        int sum = 0;
        for (int i = 0; i < TURN; i++) {
            sum += nextInt();
        }
        System.out.print(sum);
    }
}

File: 백준/Bronze/5522. 카드 게임/카드 게임.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let mut it = input.split_whitespace();
    
    let mut sum: i32 = 0;
    for i in 0..5 {
        sum += it.next().unwrap().parse::<i32>().unwrap();
    }
    print!("{}", sum);
}